nemo_text_processing/inverse_text_normalization/fr/verbalizers/decimal.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from nemo_text_processing.inverse_text_normalization.fr.graph_utils import (
+    NEMO_DIGIT,
+    NEMO_NON_BREAKING_SPACE,
+    NEMO_NOT_QUOTE,
+    GraphFst,
+    delete_space,
+)
+
+try:
+    import pynini
+    from pynini.lib import pynutil
+
+    PYNINI_AVAILABLE = True
+except (ModuleNotFoundError, ImportError):
+    PYNINI_AVAILABLE = False
+
+
+class NumberParser(GraphFst):
+    """
```

Comment:
Do you use it anywhere besides the Decimal class? Why do you need to make it a separate class?

Reply:
It's called for the integer portion of fractions. It can be written within the decimal init but this seemed to clean up clutter. Want to just merge it into decimal?

Reply:
yes, if it's used only once and doesn't need GraphFst capabilities, it should be inside the decimal class

Reply:
alright. should the same be done for the rewrite function in taggers/cardinal.py or is that fine as a separate function?

Reply:
a separate function is good